package-lock.json

```diff
@@ -16341,7 +16340,7 @@
         "@duckduckgo/autofill": {
             "version": "git+ssh://git@github.com/duckduckgo/duckduckgo-autofill.git#d29c5abc7f473b69f3d81d8a15e137ed3aab029d",
             "integrity": "sha512-RrmmFbGxMV4aKYA2ebJ6PU0IH9CW57iIYnvBemj9ZUGe8DlAheaBTOPSH3AQQk0aY7SFhNFc8UA4wBT26uM3IQ==",
-            "from": "@duckduckgo/autofill@github:duckduckgo/duckduckgo-autofill#d29c5abc7f473b69f3d81d8a15e137ed3aab029d",
```

Comment:
hm, this seems random?

Reply:
Yeah I've noticed this.

I think it's a difference with local edits of npm vs dependabot; we have had similar before but they actually caused issues.

The integrity here is the same and this commit points to the #5.0.1 so means the same thing.